pkg/kapp/resources/mod_field_copy.go

```diff
@@ -200,3 +219,50 @@ func (t FieldCopyMod) obtainValue(obj interface{}, path Path) (interface{}, bool
 
 	return obj, true, nil
 }
+
+func (t FieldCopyMod) matchRegexWithSources(path Path, srcs map[FieldCopyModSource]Resource) ([]string, error) {
+	// always looking into existing resource
```

Comment:
*question*: Why do we want to look at existing resources only?

Reply:
(Yeah so I'm also sceptical about this that's why put the comment. There was a question mark earlier but after the first iteration, I haven't got any comment on this so I removed the question mark)

Was not sure about the use case of copying from a new resource
But, I think It will be ideal to look for all the sources one by one and as soon as we got the field we can exit

Reply:
Made the changes to iterate on the given sources